Other/MergeShops/FableForestMergeShop.cs

```diff
@@ -0,0 +1,229 @@
+                    Core.RegisterQuests(3316);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.HuntMonster("fableforest", "Wind Elemental", "Wind Aura", 5);
+                        Core.HuntMonster("fableforest", "Forest Fury", "Forest Fury Feather", 5);
+                        Bot.Wait.ForPickup(req.Name);
+                    }
+                    Core.CancelRegisteredQuests();
+                    break;
+
+                case "Earth Stone":
+                    Core.FarmingLogger(req.Name, quant);
+                    Core.RegisterQuests(3317);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.HuntMonster("fableforest", "Earth Elemental", "Earth Aura", 5);
+                        Core.HuntMonster("fableforest", "Undead Satyr", "Satyr Hoof", 5);
+                        Bot.Wait.ForPickup(req.Name);
+                    }
+                    Core.CancelRegisteredQuests();
+                    break;
+
+                case "OakHeart Helm":
+                    Core.FarmingLogger(req.Name, quant);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.BuyItem("fableforest", 814, "OakHeart Helm", 1);
+                        Bot.Wait.ForPickup(req.Name);
+                    }
+                    break;
+
+                case "Fire Stone":
+                    Core.FarmingLogger(req.Name, quant);
+                    Core.RegisterQuests(3314);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.HuntMonster("fableforest", "Fire Elemental", "Fire Aura", 5);
+                        Core.HuntMonster("fableforest", "Bloodwolf", "Bloodwolf Pelt", 5);
+                        Bot.Wait.ForPickup(req.Name);
+                    }
+                    Core.CancelRegisteredQuests();
+                    break;
+
+                case "Water Stone":
+                    Core.FarmingLogger(req.Name, quant);
+                    Core.RegisterQuests(3315);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.HuntMonster("fableforest", "Water Elemental", "Water Aura", 5);
+                        Core.HuntMonster("fableforest", "Aqueevil", "Aqueevil Spirit", 5);
+                        Bot.Wait.ForPickup(req.Name);
+                    }
+                    Core.CancelRegisteredQuests();
+                    break;
+
+                case "OakHeart ArmBlades":
+                    Core.FarmingLogger(req.Name, quant);
+                    while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
+                    {
+                        Core.BuyItem("fableforest", 814, "OakHeart ArmBlades", 1);
```

Comment:
you can just do the buyline(use Adv.BuyItem if the item has *any* gold requirement, you also dont need the while if the buy item is the same as the case.

Reply:
actualy im seeing muliple buy bits so uu can combine cases *just stack them like:

case "this":
case "that":
case "another one"
case "I'm not typing anymore of these"